packages/clarity-visualize/src/visualizer.ts

```diff
@@ -98,8 +98,8 @@ export class Visualizer implements VisualizerType {
                     for (let entry of p) {
                         switch (key) {
                             case Constant.Dom:
-                                let dom = this.enrich.selectors(entry);
                                 if (entry.event === Data.Event.Discover) {
+                                    let dom = this.enrich.selectors(entry);
```

Comment:
is this an intended change?

Reply:
I thought we didn't need to create the object when we didn't use it - but it does look like enrich has some side effects so I'll revert